homeassistant/data_entry_flow.py

```diff
@@ -10,10 +10,26 @@
 
 import voluptuous as vol
 
+from .backports.enum import StrEnum
 from .core import HomeAssistant, callback
 from .exceptions import HomeAssistantError
 from .util import uuid as uuid_util
 
+
+class FlowResultType(StrEnum):
```

Comment:
I know that we normally void class names ending with `Type`, but I think here it fits and I couldn't think of a better name.

Reply:
We avoid it for "typing", but in this case, it refers to an actual type of result. So that is fine 👍